runtime/j9vm/javanextvmi.c

```diff
@@ -32,10 +36,87 @@ JVM_InitializeFromArchive(JNIEnv *env, jclass clz)
 #endif /* JAVA_SPEC_VERSION >= 11 */
 
 #if JAVA_SPEC_VERSION >= 14
+typedef struct GetStackTraceElementUserData {
+	J9ROMClass *romClass;
+	J9ROMMethod *romMethod;
+	UDATA bytecodeOffset;
+} GetStackTraceElementUserData;
+static UDATA
+getStackTraceElementIterator(J9VMThread *vmThread, void *voidUserData, UDATA bytecodeOffset, J9ROMClass *romClass, J9ROMMethod *romMethod, J9UTF8 *fileName, UDATA lineNumber, J9ClassLoader *classLoader)
+{
+	GetStackTraceElementUserData *userData = voidUserData;
+
+	/* We are done, only first stack frame is needed. */
+	userData->romClass = romClass;
+	userData->romMethod = romMethod;
+	userData->bytecodeOffset = bytecodeOffset;
+
+	return FALSE;
+}
+static void cfdumpBytecodePrintFunction(void *userData, char *format, ...)
```

Comment:
What is the purpose of this? It's controlled by a tracepoint, which is very odd.

If this was just debug, I suggest you remove it as it's going to link in unnecessary code.

Reply:
It is used by the trace point to dump the bytecode. Removing it won't compile unless the trace point is removed as well.
I can decorate the trace point along with this function with a debug define if the extra code is a concern.

Reply:
Please eliminate the tracepoint - this isn't the way to do debug.

Also please do wrap this in a debug define (I would just output the stuff always if debug is defined, no need for tracepoint).

Reply:
Sure, removed the tracepoint and wrapped it with `DEBUG_BCV` (reuse the one in `bcverify.c`).